protocol/0085-RVST-rewards_vesting.md

```diff
@@ -0,0 +1,50 @@
+# Rewards Vesting Specification
+
+## Summary
+
+The aim of the rewards vesting mechanics is to prevent farming rewards by delaying the payout of rewards through vesting. To encourage longer term behaviour parties can accelerate their rewards vesting rate through the [activity streak program](./0086-ASPR-activity_streak_program.md).
+
+## Network Parameters
+
+- `rewards.vesting.baseRate`: the proportion of rewards in a vesting account which are vested each epoch
+- `rewards.vesting.minimumTransfer`: the minimum amount (expressed in quantum) which can be vested each epoch
```

Comment:
Could we specify the default values for these in the specs?

Reply:
Sure @barnabee to confirm these are sensible but for now have set 
- `rewards.vesting.baseRate=0.1` (means ~50% of rewards are unlocked after 7 epochs and ~95% after 31 epochs)
- `rewards.vesting.minimumTransfer=100` (same as the minimum transfer network parameter).

Reply:
The minimum transfer means that someone with 1000 of rewards will be fully unlocked after 10 epochs regardless of the baserate. That seems too high.

Additionally I think the base rate ought to suggest unlocking over months rather than days for anyone with non-trivial rewards.

I think the minimum transfer amount should be something like 5 or 10, and the base rate probably in the range 1–2.5%.

**However**, I think this one we need to discuss with @MM0819 and Steven so lets hold off on confirming anything until that is done.